Project-Descriptions-and-Plans/LLVM-Toolchain/COREV_LLVM_Plan_Approved_2023.md

```diff
@@ -0,0 +1,180 @@
+| Assembler          | SIMD instructions(According to the above document, SIMD ALU Encoding, SIMD Comparison Encoding and SIMD Complex-numbers Encoding should be included)                                                                                                              | 2023-03-30    | ISCAS       |
+| Assembler          | Bit Manipulation Encoding, Event Load Instructions, etc. This time should contain all assembly instructions.                       | 2023-04-30    | ISCAS       |
+| Compiler options   | CV32E40Pv2 has its own compilation options, support it. (xcvhwlp,xcvsimd,xcvbi,xcvmem,xcvalu,xcvmac,xcvbitmanip,xcvelw )                                                 | 2023-05-30    | ISCAS       |
+| Codegen            | SIMD builtins(Please see corev-builtin-spec.md for more details)                                                               | 2023-07-30    | ISCAS       |
+| Codegen            | Bitmanip builtins(Please see corev-builtin-spec.md for more details)                                                           | 2023-09-30    | ISCAS       |
+| Codegen            | ALU builtins(Please see corev-builtin-spec.md for more details)                                                                | 2023-10-30    | ISCAS       |
+| Codegen            | Post-indexed and register-indexed memory access builtins. This time should contain all builtins.                               | 2023-11-30    | ISCAS       |
+| Hardware loop      | CV32E40Pv1 has already implemented version 1, and new features will be added to it later                                       | 2023-12-30    | ISCAS       |
+
+### Project tracking and meetings
+
+The progress towards 2023 milestones will be tracked in progress meetings. Slides will be updated during the meeting and posted on LLVM Tools and Kanban Board Mattermost channels.
+
+The various activities are led in a unified project way and reported to the relevant task groups.
+
+The Corev-llvm-project meets every week.
+
+### Risk register
+
+|                           | Likelihood (1-10) | Impact (1-3) | Avoidance&nbsp;/ Mitigation                                                   |
+| ------------------------- | :---------------: | :----------: | ----------------------------------------------------------------------------- |
+| Not enough resources      | 8                 | 3            | 24 More technical participants are welcome. Prioritize most important features first.                                       |
+| Insufficient coordination | 5                 | 2            | 10 Weekly meetings                                                               |
+| Conflicting contributions | 5                 | 2            | 10 Weekly meetings                                                               |
```

Comment:
You are missing a column in the table.  You also need a bit more on the mitigation. Something like:
```
|   Risk                    | Likelihood (1-10) | Impact (1-3) | Overall | Mitigation                                                                         |
| :------------------------ | :---------------: | :----------: | :-----: | :--------------------------------------------------------------------------------- |
| Not enough resources      | 8                 | 3            | 24      | More technical participants are welcome. Prioritize most important features first. |
| Insufficient coordination | 5                 | 2            | 10      | Weekly meetings  to coordinate work                                                |
| Conflicting contributions | 5                 | 2            | 10      | Weekly meetings  to schedule different contributions                               |
```

Reply:
done